src/openforms/registrations/contrib/objects_api/tests/test_backend_v2.py

```diff
@@ -168,6 +171,157 @@ def test_submission_with_objects_api_v2(self):
             },
         )
 
+    def test_submission_with_objects_api_v2_with_payment_attributes(self):
+        submission = SubmissionFactory.from_components(
+            [
+                # fmt: off
+                {
+                    "key": "age",
+                    "type": "number"
+                },
+                {
+                    "key": "lastname",
+                    "type": "textfield",
+                },
+                {
+                    "key": "location",
+                    "type": "map",
+                },
```

Comment:
please drop all this setup/configuration that is not relevant for the thing that we're testing. form fields / location etc. are only distraction from what is actually being tested